contracts/governance/GovernorAlpha.sol

```diff
@@ -44,6 +44,18 @@ contract GovernorAlpha is SafeMath96 {
 		return 2880;
 	} // ~1 day in blocks (assuming 30s blocks)
 
+	/// @notice Threshold for total votes proposal to be able to cancelled by guardian.
+	/// TODO finalized this value
+	function totalVotesForCancellationThreshold() public pure returns (uint256) {
+		return 50000000;
+	}
+
+	/// @notice Threshold for total participant of the proposal to be able to cancelled by guardian.
+	/// TODO finalized this value
+	function participantForCancellationThreshold() public pure returns (uint256) {
+		return 50000000;
+	}
+
```

Comment:
anything speaking against making it constant? ;o)

Reply:
The contract is not upgradeable.
And also, the value is kind of rare to be changed.
So in my opinion, a constant is one of the best solutions here, besides it saves the gas as well.